graph-apps/graph-apps.adoc

```diff
@@ -0,0 +1,675 @@
+
+. Basic knowledge of HTML and JavaScript and an understanding of Single Page Applications (SPAs)
+. Knowledge of the https://neo4j.com/developer/javascript[offical JavaScript Drivers]
+. If you haven't already, http://neo4j.org/download[download and install Neo4j Desktop^].
+
+
+== Introduction
+
+So, you would like to build a Graph App?  You are in good company, many enterprises and community developers are building Graph Apps for a wide range of use cases.
+These range from utility tools like monitoring tool https://halin.graphapp.io/[Halin^] and educational Graph Apps like Neuler to visulisation apps including https://neo4j.com/blog/graphxr-graph-app-neo4j-desktop/[GraphXR by Kineviz] and https://medium.com/neo4j/introducing-neomap-a-neo4j-desktop-application-for-spatial-data-3e14aad59db2[NeoMap] built by community member Estelle Scifo.
+
+Graph Apps are a convenient way of trying out ideas or building useful tools to help manage Neo4j databases.  You could take an existing SPA and package it into a graph App or start from scratch with a new idea.
+
+If you are looking for inspiration, https://install.graphapp.io/[head over to the Graph App Gallery^] to see what other developers have built.
+Otherwise, let's get started.
+
+
+== What are Graph Apps?
+
+A Graph App is a Single Page Application built with HTML and JavaScript which interact with Neo4j databases through https://neo4j.com/desktop/[Neo4j Desktop^].
+These can be built with vanilla HTML and JavaScript, but most commonly we see these built with modern front end frameworks like https://reactjs.org/[React^] or https://vuejs.org/[Vue.js^].
+
+Whichever framework you choose to build your application, you will interact with the Graph the same way; by running a Cypher Query through the https://github.com/neo4j/neo4j-javascript-driver[Official JavaScript drivers].
+There is plenty of content over on the https://neo4j.com/developer/javascript/[JavaScript Developer Pages] so we will not cover this in too much detail, only the nuances around how the driver works with the Neo4j Desktop API.
+
+
+== Enabling Developer Tools
+
+Neo4j Desktop comes with a Development App that can be used while developing and debugging your Graph App.
+To enable Development Mode, click on the Settings image:{img}settings.png[Settings Icon, title="Settings Icon"] icon in the bottom left hand corner of the screen to reveal the Settings pane.
+
+At the bottom of the Settings pane, you will see a *Developer Tools* form.
+Click *Enable Development Mode* and add fill out the Entry Point and Root Path.
+
+image:{img}developer-tools.png[Developer Tools window with Enable development mode checkbox and fields for Entry Point and Root Path, title="Developer Tools form"]
+
+- *Entry Point*: This is the path to your app, either a location of a HTML file (for example `file:///Users/me/my-graph-app/index.html`) or a HTTP url where your app is being served (for example `http://localhost:8080`).
```

Comment:
did the file:// thing work?

Reply:
Not in the latest version but it's listed on graph-app-starter so it must have been a valid option at some point.  There's a card in trello